src/lib/Form/Data/ContentTypeData.php

```diff
@@ -40,6 +42,24 @@ class ContentTypeData extends ContentTypeUpdateStruct implements NewnessCheckabl
     /** @var \Ibexa\Contracts\Core\Repository\Values\ContentType\ContentTypeDraft */
     protected $contentTypeDraft;
 
+    private bool $new;
+
+    public function __construct(array $properties = [])
```

Comment:
Or something more strict:
```suggestion
    /**
     * @param array<mixed> $properties
     */
    public function __construct(array $properties = [])
```